### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by dzmitry-duboyski**
### What's done:
- Fix "plugin" => "extension" in README.md
- Fix "плагин" => "расширение" in README.ru.md
- Fix link to acticle on the blog in README.ru.md
- Updated the title for the article in README.md
- Updated header:
   usage-2captcha-solver-in-puppeteer => Usage 2captcha-solver extension in puppeteer
   usage-2captcha-solver-in-puppeteer => Использование расширения 2captcha-solver в puppeteer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 [English](README.md) | [Русский](README.ru.md)
-# usage-2captcha-solver-in-puppeteer
+# Usage 2captcha-solver extension in puppeteer
```
Comment (kratzky) [inline README.md]: usage of | how to use

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 The example uses [Puppeteer](https://pptr.dev/), [puppeteer-extra](https://www.npmjs.com/package/puppeteer-extra) and [puppeteer-extra-plugin-stealth](https://www.npmjs.com/package/puppeteer-extra-plugin-stealth). [puppeteer-extra](https://www.npmjs.com/package/puppeteer-extra) is a lightweight wrapper around Puppeteer and [puppeteer-extra-plugin-stealth](https://www.npmjs.com/package/puppeteer-extra-plugin-stealth) is an add-on to [puppeteer-extra](https://www.npmjs.com/package/puppeteer-extra) designed to hide the traces of automation.
 
-A detailed description of how this example works is described in our article [How to use the 2captcha-solver plugin in Puppeteer](https://2captcha.com/blog/how-to-use-2captcha-solver-extension-in-puppeteer).
+A detailed description of how this example works is described in our article [How to use the 2Captcha solver extension in Puppeteer for bypass reCAPTCHA, hCaptcha and others captchas](https://2captcha.com/blog/how-to-use-2captcha-solver-extension-in-puppeteer).
```
Comment (kratzky) [inline README.md]: description described?

> A detailed description of how this example works can be found in...

**Review by kratzky — CHANGES_REQUESTED** [review]
just update a couple of phrases :)

File: README.md (hunk 2 above)
Comment (dzmitry-duboyski) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (dzmitry-duboyski) [reply README.md]: fixed

**Pull request merged**